orc8r-nginx-operator/tests/unit/test_charm.py

```diff
@@ -466,25 +460,18 @@ def test_given_container_can_be_connected_when_rootca_certificate_available_then
             path="/var/opt/magma/certs/rootCA.pem", source=test_rootca_cert
         )
 
-    @patch("ops.model.Container.exec")
+    @patch("ops.model.Container.push")
     def test_given_valid_domain_config_set_when_config_changed_then_nginx_config_file_is_created(
```

Comment:
```suggestion
    def test_given_valid_domain_config_set_when_config_changed_then_nginx_config_file_is_pushed(
```
netpicking, but maybe it's better to justify the usage of `patched_push` here :)